akka-http-core/src/main/java/akka/http/javadsl/model/HttpEntities.java

```diff
@@ -42,6 +48,26 @@ private HttpEntities() {}
         return HttpEntity$.MODULE$.apply((akka.http.scaladsl.model.ContentType) contentType, bytes);
     }
 
+    public static Multipart.FormData fromParts(Multipart.FormData.BodyPart ... parts) {
+        return akka.http.scaladsl.model.Multipart.FormData$.MODULE$.applyNonStrict(convertArray(parts));
+    }
+
+    public static Multipart.FormData.Strict fromParts(Multipart.FormData.BodyPart.Strict ... parts) {
+        return akka.http.scaladsl.model.Multipart.FormData$.MODULE$.applyStrict(convertArray(parts));
+    }
+
+    public static Multipart.FormData.BodyPart create(String name, BodyPartEntity entity, Map<String, String> additionalDispositionParams) {
+        List nil = Nil$.MODULE$;
+        return akka.http.scaladsl.model.Multipart$FormData$BodyPart$.MODULE$.apply(name, (akka.http.scaladsl.model.BodyPartEntity) entity,
+                convertMapToScala(additionalDispositionParams), nil);
```

Comment:
Already tried this. Unfortunately, gives a compile error. So I'd rather leave as is. List import looks confusing, but so do too complex object invocations like `Multipart$FormData$BodyPart$`. Which render "red" in IntelliJ. However this is some internal code, not for external consumption.